src/main/java/org/akip/domain/DecisionDefinition.java

```diff
@@ -0,0 +1,92 @@
+package org.akip.domain;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.hibernate.annotations.NaturalId;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "decision_definition")
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public class DecisionDefinition implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    private Long id;
+
+    @NaturalId
+    @Column(name = "dmn_decision_definition_id")
+    private String dmnDecisionDefinitionId;
+
+    @Column(name = "name")
+    private String name;
+
+    @OneToMany(mappedBy = "decisionDefinition")
```

Comment:
Avoid this dependency. Why does the decisionDefinitiion need the deployments? You can remove it.